### ISSUE TITLE
gosnappi GRPC setConfig is failing against ixNetwork

### ISSUE THREAD

**Issue by ANISH-GOTTAPU** [open-traffic-generator/snappi-ixnetwork#439]
**Version information**
snappi[ixnetwork]==0.5.3

**Describe the bug**
gosnappi GRPC setConfig is failing against ixNetwork

**To Reproduce**
1) Create and run grpc server with ixnetwork as target-host
2) Create a test using gosnappi which uses grpc transport and sets the config on ixNetwork.
3) Make sure the test consists of atleast one flow creation.

**Expected behavior**
1) Shouldn't see any errors and config should apply properly

**Additional context**

automation@automation-virtual-machine:~/otg-grpc/logs$ cat grpc_server-2021-09-07-08-03-20.log
2021-09-07 13:33:20,433 : Starting gRPC Server [OTG API Version = 0.4.10] ...
2021-09-07 13:33:20,435 : Enabling insecure channel ...
2021-09-07 13:33:20,435 : Enabled insecure channel
2021-09-07 13:33:20,436 : gRPC Server Started at [::]:40051 ...
2021-09-07 13:33:32,116 : Snappi Remote Server address = 10.39.71.172:11009 in IxNetwork mode
2021-09-07 13:33:32,116 : Snappi initialized
2021-09-07 13:33:32,116 : Requesting set_config ...
2021-09-07 13:33:34,486 : Snappi_api Setconfig Returned Exception :    File "/home/automation/.local/lib/python3.8/site-packages/snappi_ixnetwork/snappi_api.py", line 268, in set_config
    self.config_ixnetwork(config)
  File "/home/automation/.local/lib/python3.8/site-packages/snappi_ixnetwork/snappi_api.py", line 317, in config_ixnetwork
    self.traffic_item.config()
  File "/home/automation/.local/lib/python3.8/site-packages/snappi_ixnetwork/trafficitem.py", line 647, in config
    self.copy_flow_packet(self._config)
  File "/home/automation/.local/lib/python3.8/site-packages/snappi_ixnetwork/trafficitem.py", line 625, in copy_flow_packet
    parent = pkt.parent.__deepcopy__(None)

**Comment by ANISH-GOTTAPU** [comment]
```package gosnappi_test

import (
	log "log"
	"testing"

	"github.com/open-traffic-generator/snappi/gosnappi"
)

var grpcServerLocation = "10.39.71.143:40051"

func b2bRawConfig(api gosnappi.GosnappiApi) gosnappi.Config {
	cfg := api.NewConfig()
	tx := cfg.Ports().Add()
	tx.SetName("port1").SetLocation("otg-novus100g.lbj.is.keysight.com;1;1")
	rx := cfg.Ports().Add()
	rx.SetName("port2").SetLocation("otg-novus100g.lbj.is.keysight.com;1;2")
	f1 := cfg.Flows().Add().SetName("f1")
	f1.TxRx().Port().SetTxName(tx.Name()).SetRxName(rx.Name())
	return cfg
}

func TestUdpFixed(t *testing.T) {
	api := gosnappi.NewApi()
	api.NewGrpcTransport().SetLocation(grpcServerLocation)

	cfg := b2bRawConfig(api)
	f1 := cfg.Flows().Items()[0]

	eth := f1.Packet().Add().Ethernet()
	eth.Src().SetValue("a2:71:d4:59:b3:b8")
	eth.Dst().SetValue("6e:d2:47:65:aa:69")

	ipv4 := f1.Packet().Add().Ipv4()
	ipv4.Src().SetValue("10.10.10.1")
	ipv4.Dst().SetValue("10.10.10.2")

	udp := f1.Packet().Add().Udp()
	udp.SrcPort().SetValue(3000)
	udp.DstPort().SetValue(4000)

	// f1.Metrics().SetEnable(true)
	f1.Duration().FixedPackets().SetPackets(10000)
	f1.Size().SetFixed(74)
	resp, _ := api.SetConfig(cfg)
	log.Print(resp)
}```

**Comment by ANISH-GOTTAPU** [comment]
Fixed in Snappi version snappi[ixnetwork]==0.5.4
